src/main/scala/com/databricks/spark/sql/perf/results.scala

```diff
@@ -77,10 +80,12 @@ case class BenchmarkResult(
     result: Option[Long] = None,
     breakDown: Seq[BreakdownResult] = Nil,
     queryExecution: Option[String] = None,
-    failure: Option[Failure] = None)
+    failure: Option[Failure] = None,
+    ml: Option[MLResult] = None)
```

Comment:
I moved the ml test params to `BenchmarkResult`. Note that the regular sql parameters, which are an empty map in the code base (???), are stored separately in `parameters`.